app/routes/planets_route.py

```diff
@@ -0,0 +1,120 @@
+from app import db
+from app.models import planet
+from app.models.planet import Planet
+from flask import Blueprint, jsonify, make_response, request, abort
+
+
+planets_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+
+''' Helper functions - check if is_valid_integer'''
+
+def valid_int(number, parameter_type):
+    try:
+        int(number)
+    except:
+        abort(make_response({"error": f'{parameter_type} must be an integer'}, 400))
+
+
+def get_planet_from_id(planet_id):
+    valid_int(planet_id, "planet_id")
+    return Planet.query.get_or_404(planet_id, description='{Planet not found}')
```

Comment:
`get_or_404` is a great method to either get a instance of the planet or return a 404. In the case where it's not a `404`, we want to use the instance of the `planet` to return an appropriate response body. Here's one way we can refactor to return an appropriate response body:
```suggestion
    planet =  Planet.query.get_or_404(planet_id, description='{Planet not found}')
    return planet.to_dict()
```